app/src/main/java/com/ttbmp/cinehub/app/usecase/manageemployeesshift/request/GetShiftListRequest.java

```diff
@@ -41,7 +41,7 @@ public void setStart(LocalDate start) {
 
     @Override
     protected void onValidate() {
-        if (cinema == null) {
+        if (cinemaId == -1) {
```

Comment:
Is there any use case justifying the hassle of comply with this rule in our persistence implementation?